k8s/solr/README.md

```diff
@@ -0,0 +1,417 @@
+# Overview
+
+Apache Solr is an open-source enterprise-search platform, written in Java, from the Apache Lucene project.
+
+For more information on Apache Solr, see the Apache Solr [official website](https://lucene.apache.org/solr/).
+
+## About Google Click to Deploy
+
+Popular open stacks on Kubernetes, packaged by Google.
+
+## Architecture
+
+![Architecture diagram](resources/apache-solr-k8s-app-architecture.png)
+
+This solution is focused on SolrCloud mode which will use ZooKeeper as external repository. By default 3 replicas of Solr nodes and 3 replicas of ZooKeeper nodes will be deployed. For more information about Solrcloud visit [official documentation](https://lucene.apache.org/solr/guide/8_6/solrcloud.html).
```

Comment:
SolrCloud

Reply:
it is Solr node, not SolrCloud node, or you mean somewhere else exactly?

Reply:
Better to use present tense everywhere
This solution is focused on SolrCloud mode which **uses** ZooKeeper as external repository. By default 3 replicas of Solr nodes and 3 replicas of ZooKeeper nodes **are** deployed. For more information about **SolrCloud** visit [official documentation](https://lucene.apache.org/solr/guide/8_6/solrcloud.html).

Btw. is there a documentation link independent of version?